src/main/java/org/testng/ClassMethodMap.java

```diff
@@ -17,26 +17,26 @@
  * @author <a href='mailto:the[dot]mindstorm[at]gmail[dot]com'>Alex Popescu</a>
  */
 public class ClassMethodMap {
-  private Map<Object, List<ITestNGMethod>> m_classMap = Maps.newHashMap();
+  private Map<Object, List<ITestNGMethod>> classMap = Maps.newHashMap();
   // These two variables are used throughout the workers to keep track
   // of what beforeClass/afterClass methods have been invoked
-  private Map<ITestClass, Set<Object>> m_beforeClassMethods = Maps.newHashMap();
-  private Map<ITestClass, Set<Object>> m_afterClassMethods = Maps.newHashMap();
+  private Map<ITestClass, Set<Object>> beforeClassMethods = Maps.newHashMap();
```

Comment:
@juherr Lol.. Yeah you were right on that... I got a bit carried away and obsessed with driving down the sonar cube count down... I must admit, its a bit difficult to keep sonar cube happy :)